build/crypto.m4

```diff
@@ -287,3 +287,28 @@ AC_DEFUN([TS_CHECK_CRYPTO_SET_CIPHERSUITES], [
   TS_ARG_ENABLE_VAR([use], [tls-set-ciphersuites])
   AC_SUBST(use_tls_set_ciphersuites)
 ])
+
+dnl
+dnl Since OpenSSL 1.1.1
+dnl
+AC_DEFUN([TS_CHECK_EARLY_DATA], [
+  AC_MSG_CHECKING([OpenSSL early data support])
+  AC_TRY_RUN([
+#include <openssl/opensslv.h>
+int main() {
+  if (OPENSSL_VERSION_NUMBER >= 0x10101000L) {
```

Comment:
`OPENSSL_VERSION_NUMBER` is going to be deprecated on OpenSSL 3.0.0.
Why don't you check availability of `SSL_set_max_early_data`?

Reply:
Good suggestion. If it's easy to do, then I say do this "right" now, rather than adding more stuff to our old legacy code. If complicated, maybe defer until later when we have to cleanup all checks on OPENSSL_VERSION_NUMBER.

Reply:
fixed